app/controlplane/api/workflowcontract/v1/crafting_schema.proto

```diff
@@ -84,6 +84,10 @@ message CraftingSchema {
       // https://github.com/microsoft/sarif-tutorials/blob/main/docs/1-Introduction.md
       SARIF = 9;
       HELM_CHART = 10;
+
+      // Evidences are materials that are not meant to be subject of attestations, but still uploaded
```

Comment:
I believe that `evidences` (plural) is not a correct term, I'd mention it here `Pieces Of Evidences`.

Also, I do not think that the subject part is the important attribute of it but actually what's for, so I'd update the comment to something like this

> Pieces of evidences represent generic, additional context that doesn't fit into one of the well known material types. For example, a custom approval report (in json), ...`

What do you think?